api-docs/rst/dev-guide/concepts.rst

```diff
@@ -105,6 +105,12 @@ The launch configuration specifies the launch type along with server and load ba
                 if you use a load balancer so the load balancer can retrieve the IP address of new
                 servers. See :ref:`Load balancer parameters <load-balancers-parameters>`.
 
+        ``draining_timeout``
+             Specifies the number of seconds autoscale will put the CLB node in DRAINING
+             before deleting the node and eventually the server. This is used when scaling down.
```

Comment:
just a minor change from autoscale to "Autoscale" for consistency. Also please add "mode" after DRAINING